src/main/java/core/basesyntax/AreaCalculationInterface.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculationInterface {
```

Comment:
The interface name `AreaCalculationInterface` is not following Java naming conventions. Avoid using verbs for class/interface names. Consider renaming it to something like `AreaCalculator` to make it more intuitive and in line with Java practices.

Reply:
read this and fix